src/common/Button/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+export const Button = (props) => {
+	const handleClick = (e) => {
+		if (props.onClick) {
+			e.preventDefault();
+			props.onClick();
```

Comment:
pass here e because when createAuthor method is called it expected to have event as an argument, but it receives undefined